static/app/views/alerts/rules/details/body.tsx

```diff
@@ -346,57 +336,55 @@ export default class DetailsBody extends React.Component<Props> {
                 )}
               <StyledLayoutBodyWrapper>
                 <Layout.Main>
-                  <HeaderContainer>
-                    <HeaderGrid>
+                  <HeaderGrid>
+                    <HeaderItem>
+                      <Heading noMargin>{t('Display')}</Heading>
+                      <ChartControls>
+                        <DropdownControl
+                          label={
+                            getDynamicText({
+                              fixed: 'Oct 14, 2:56 PM — Oct 14, 4:55 PM',
+                              value: timePeriod.display,
+                            }) ?? '' // we should never get here because timePeriod.display is typed as always defined
+                          }
+                        >
+                          {TIME_OPTIONS.map(({label, value}) => (
+                            <DropdownItem
+                              key={value}
+                              eventKey={value}
+                              isActive={!timePeriod.custom && timePeriod.period === value}
+                              onSelect={this.props.handleTimePeriodChange}
```

Comment:
should we destructure this prop? https://github.com/getsentry/sentry/blob/5a58dc579d78ff745229a487662d5fe87e07e4b2/static/app/views/alerts/rules/details/body.tsx#L312